database/mysql/seed.js

```diff
@@ -0,0 +1,43 @@
+const data = require('./data');
+
+const productCount = 100000;
```

Comment:
is this just for testing? shouldn't this be 10 million?

Reply:
yes, this becomes 10 million on the next PR